lib/logflare/endpoints/query.ex

```diff
@@ -81,7 +84,8 @@ defmodule Logflare.Endpoints.Query do
       :max_limit,
       :enable_auth,
       :language,
-      :description
+      :description,
+      :labels
```

Comment:
maybe worth adding some basic validations? For example when setting `labels: "field1,"` I think the trailing comma could make it so there are two labels, `"field1"` and `""`. But yeah, if that's not gonna break anything I probably wouldn't bother with extra validations.

Reply:
nice one, yea trailing commas would add an unnecessary empty label for it, added some filtering for that